core/src/components/tooltip/tooltip.stories.tsx

```diff
@@ -128,7 +128,7 @@ const ComponentTooltip = ({ tooltipPosition, mouseOverTooltip, text, slot }) =>
     </sdds-tooltip>
 
     <!-- Demo button for presentation purposes -->
-    <sdds-button size= 'sm' id="button-1" text='Hover me'></sdds-button>
+    tds-button size= 'sm' id="button-1" text='Hover me'></tds-button>
```

Comment:
```suggestion
    <tds-button size= 'sm' id="button-1" text='Hover me'></tds-button>
```
Missing opening tag "<".